.travis.yml

```diff
@@ -29,8 +29,10 @@ node_js:
 
 ## Use this to prepare your build for testing
 ## e.g. copy database configurations, environment variables, etc.
-# before_script:
-#
+## http://stackoverflow.com/questions/12336566/travis-ci-with-jasmine-node
+before_script:
```

Comment:
would including multiple items per section be of use/worth a try?
http://docs.travis-ci.com/user/build-configuration/